models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md

```diff
@@ -0,0 +1,97 @@
+# person-vehicle-bike-detection-crossroad-1020
+
+## Use Case and High-Level Description
+
+This is a YOLO V3 network finetuned for Person/Vehicle/Bike detection for security
+surveillance applications. Works in a variety of scenes and weather/lighting
+conditions.
+
+Yolo V3 is a real-time object detection model implemented with Keras\*
+from this [repository](https://github.com/david8862/keras-YOLOv3-model-set)
+and converted to TensorFlow\* framework.
+
+This model was pretrained on COCO\* dataset with 80 classes and then finetuned for
+Person/Vehicle/Bike detection.
+
+
+## Example
+
+![](./person-vehicle-bike-detection-crossroad-1020.png)
+
+## Specification
+
+| Metric                          | Value       |
+|---------------------------------|-------------|
+| Mean Average Precision (mAP)    | 48.89%      |
+| AP people                       | 58.94%      |
+| AP vehicles                     | 62.05%      |
+| AP bikes/motorcycles            | 25.66%      |
+| Max objects to detect           | 200         |
```

Comment:
Where did this parameter come from? Original Yolo can produce much more detections.

Reply:
Please, note that we evaluate on our crossroad test dataset (not COCO dataset)  
This is written below, where we describe the train/test datasets.

The original model has much worse result on our crossroad dataset.

Reply:
Nevertheless, I don't understand why you set max objects to 200. According to output description, it produces much more. Could you explain what it mean?

Reply:
I removed the line

Indeed, dataset contains not so many objects with height > 200, so this is less reliable, but nevertheless they may be detected